packages/blockchain-api/src/blockscout.ts

```
@@ -27,6 +28,18 @@ export interface BlockscoutCeloTransfer {
   value: string
 }
 
+// Calculate cGLD to cUSD rate based on the actual transfers
+function getKnownExchangeRates(
+  current: BlockscoutCeloTransfer,
+  comparable: BlockscoutCeloTransfer
+) {
+  return current.token === CGLD && comparable.token === CUSD
```

If current must be cGLD and comparable must be cUSD, could this be enforced in the function signature, or throw an error rather than returning null?